main.js

```diff
@@ -0,0 +1,298 @@
+          playerStatus.innerHTML = currentPlayer.name + ' wins!'
+          delayEnding()
+        } else if (fullGrid.length === 9) {
+          playerStatus.innerHTML = "It's a draw!"
+          delayEnding()
+        }
+        else {
+          currentPlayer = playerX
+          playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+        }
+
+      }
+    }
+  }
+}
+
+// ------------------------------------------------------------------------------------------------------------
+// one-player game between human and computer
+
+// starts game, disabling both player option buttons 
+
+function startGame1() {
+  playerX.name = "Player X"
+  playerO.name = "Player O"
+  currentPlayer = playerX
+  playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+  event.target.disabled = true
+  startButton2.disabled = true
+  onePlayerGame()
+}
+
+// random number generator to choose among the available spaces in the board array (by index)
+
+function randomInt(max, min) {
+  max = board.length - 1
+  min = 0
+  arrayIndex = Math.floor(min + (Math.random() * (max - min + 1)))
+  return arrayIndex
+}
+
+// main game function 
+
+board = [c0, c1, c2, c3, c4, c5, c6, c7, c8]
+
+function onePlayerGame() {
+
+  for (let square of cells) {
+    square.addEventListener('click', playGame)
+
+    // functions that use the "square" variable which (due to scope) cannot be defined outside of the onePlayerGame
+    
+    // function to eliminate unplayable spaces from the board 
+    
+    function eliminateCells() {
+      if (board.includes(square)) {
+        let indexOfItem = board.indexOf(square)
+        board.splice(indexOfItem, 1)
+        return board
+      }
+    }
+
+    // computer's move that puts an O in the HTML and removes the computer's move from the board array
+
+    function computerMove() {
+      square = board[arrayIndex]
+      square.innerHTML = currentPlayer.symbol
+      board.splice(arrayIndex, 1)
+      return board
+    }
+
+    function playGame() {
+      // human's part of the game -----------------
+
+      // prevents clicking on a filled square by throwing alert
+
+      if (square.innerHTML !== "") {
+        window.alert("Please choose an empty square!")
+      } else {
+        square.innerHTML = currentPlayer.symbol
+        console.log(currentPlayer.symbol)
+      }
+
+      // checks for tie       
+
+      tieGame()
+
+      // after human has selected a cell
+
+      if (square.innerHTML === "X") {
+
+        gameWin()
+        eliminateCells()
+
+        if (checkedMoves.length === 3) {
+          playerStatus.innerHTML = currentPlayer.name + " wins!"
+          delayEnding()
+        } else if (fullGrid.length === 9) {
+          playerStatus.innerHTML = "It's a draw!"
+          delayEnding()
+        } else {
+          currentPlayer = playerO
+          playerStatus.innerHTML = currentPlayer.name + "'s Turn";
+
+          // computer's part of the game -----------------------
```

Comment:
You could use a timer here to delay the computer making a move if you wanted the game to play more like there is an actual person playing for the computer, similar to what you have to delay the ending of the game and reload the page